public/app/routes/routes.tsx

```diff
@@ -207,18 +207,33 @@ export function getAppRoutes(): RouteDescriptor[] {
     },
     {
       path: '/org/teams',
-      roles: () => (config.editorsCanAdmin ? [] : ['Editor', 'Admin']),
+      // TODO why the legacy condition is: "editorsCanAdmin ? [] : ['Editor', 'Admin']"
+      // instead of:                       "editorsCanAdmin ? ['Editor', 'Admin'] : ['Admin']""
+      roles: () =>
+        contextSrv.evaluatePermission(() => (config.editorsCanAdmin ? [] : ['Editor', 'Admin']), [
+          AccessControlAction.ActionTeamsRead,
+        ]),
       component: SafeDynamicImport(() => import(/* webpackChunkName: "TeamList" */ 'app/features/teams/TeamList')),
     },
     {
       path: '/org/teams/new',
```

Comment:
hmm, is this a pattern that we kinda follow in grafana? 🤔 or is this different in different sections?

just curious, it might just be that whatever works >D